### PR TITLE
Add installer test

### PR DESCRIPTION
**Pull request by justafish**
* [x] Ready for review
* [ ] Ready for merge

This adds a Nightwatch test for making sure the installer works

### Notes

- Splits the installer into two scripts, one for installation and one for running the web server
- Moves https://github.com/contentacms/contenta_jsonapi_project into this repo because `composer create-project` has some "quirks" and doesn't always do the equivalent of checking out another project and then installing it

I think it might be worth removing the quick/remote install as it's almost as many steps as the regular install now.

I know that we also have CircleCI, but I'm much more familiar with Travis and was able to get it working on there more easily. I think it's ok to have multiple CIs!

### Test Instructions

Make sure Travis is green!

### QA
 https://travis-ci.org/justafish/contenta_jsonapi/builds/247126337

### CODE REVIEW

**Comment by e0ipso** [comment]
Cool! Let's get this in, we can refactor things later if needed to use a sing CI.

**Comment by e0ipso** [comment]
@justafish can you post a summary of the changes required for a user trying the quick install?

**Comment by justafish** [comment]
@e0ipso I've updated the README to reflect all the changes in this PR, there's no changes for a user running the quick install - I'm just not sure it's necessary now!

Quick install:
```
php -r "readfile('https://raw.githubusercontent.com/contentacms/contenta_jsonapi/8.x-1.x/installer.sh');" > contentacms.sh
chmod a+x contentacms.sh
./contentacms.sh
```

Normal install:
```
$ git clone git@github.com:contentacms/contenta_jsonapi.git
$ cd contenta_jsonapi
$ composer run-script install-contenta /var/www/contenta --timeout=0
$ composer run-script start-contenta /var/www/contenta --timeout=0
```

File: README.md [hunk 1]
```diff
@@ -1,27 +1,37 @@
 # Contenta
 
-## Quick Install
+## Requirements
 
 - Install [composer](https://getcomposer.org/)
 - Make sure you have the sqlite extension for PHP (if you're using the default install on a mac, this should already be there)
```
Comment (dawehner) [inline README.md:6]: As a follow up, let's move this requirement to the quick install ... the normal installation doesn't actually require sqlite, anyway ...

**Comment by e0ipso** [comment]
Let's create an `installer.sh` that replicates the same than we had before by calling the commands in https://github.com/contentacms/contenta_jsonapi/pull/73#issuecomment-311085088

**Comment by justafish** [comment]
@e0ipso I don't understand what you're asking, could you clarify? This PR maintains an `installer.sh` that does exactly that, as I mentioned in my previous comment.
I suggested removing it because it's not any easier or quicker than performing the other method of installation.

**Comment by dawehner** [comment]
The reason we have a separate project ```contenta_jsonapi_project``` is that one wants to separate the repository of contenta from the code everyone would store for their own project. ```contenta_jsonapi_project```  pulls in contenta, but it also provides a base for people to require other modules etc. 
We need a way to pull in contenta separately, as this is the only way how we can run on hosters, which require their old structure (pantheon, acquia and platform).

I try to wrap my head around how this would work with a single repository. At least for me the readme doesn't explain that. These scripts are, at least for me, just for local development purposes.

**Comment by justafish** [comment]
> one wants to separate the repository of contenta from the code everyone would store for their own project

This PR does that. You provide it with your destination directory and it installs it into there, same as previously.

**Comment by justafish** [comment]
Moving to #74

**Pull request closed without merging**